duplicacy-util.go

```diff
@@ -419,8 +419,14 @@ func performBackup() error {
 			cmdArgs := []string{"backup", "-storage", backupInfo["name"], "-threads", backupInfo["threads"], "-stats"}
 			if backupInfo["vss"] == "true" {
 				cmdArgs = append(cmdArgs, "-vss")
-				logMessage(logger, fmt.Sprint("Backing up to storage ", backupInfo["name"],
-					" -vss with ", backupInfo["threads"], " threads"))
+				if backupInfo["vssTimeout"] != "" {
+					cmdArgs = append(cmdArgs, "-vss-timeout", backupInfo["vssTimeout"])
+					logMessage(logger, fmt.Sprint("Backing up to storage ", backupInfo["name"],
+						" -vss -vss-timeout ", backupInfo["vssTimeout"],  " with ", backupInfo["threads"], " threads"))
+				} else {
+					logMessage(logger, fmt.Sprint("Backing up to storage ", backupInfo["name"],
+						" -vss with ", backupInfo["threads"], " threads"))
+				}
```

Comment:
This was a touch sloppy when I implemented it, and now it really shows. I see that you continued what I had done, which is great, but not when what I had done kinda sucks ... :smile:

Excuse typos, not checking for syntax. Could you change this to be something like:

```
vssFlags := ""
if backupInfo["vss"] == "true" {
    cmdArgs = append(cmdArgs, "-vss")
    vssFlags = " -vss"
   if backupInfo["vssTimeout"] != "" {
    cmdArgs = append(cmdArgs ...)
    vssFlags := fmt.Sprintf("%s -vss-timeout %s", vssFlags, backupInfo[vssTimeout"]
  }
```

Now you can have one LogMessage for "Backup up to storage", which I think is cleaner.